### ISSUE TITLE
Invalid swagger yaml on procedures with empty arg/ret

### ISSUE THREAD

**Issue by shlomi-nx** [nutanix/papiea-js#134]
If no argument/return shapes are defined for procedures (or defined as nothing `{}`), we get invalid swagger yaml:

```yaml
    post:
      description: Calls a procedure ping
      operationId: callxi_clustersping
      tags:
        - xi_clusters/0.0.1/imaged_host/procedure
      parameters:
        - name: uuid
          in: path
          description: UUID of the entity
          required: true
          schema:
            type: string
            format: uuid
      requestBody:
        description: ping input
        required: true
        content:
          application/json:
            schema:
              properties:
                input:
                  $ref: '#/components/schemas/undefined'    # <=
      responses:
        '200':
          description: ping response
          content:
            application/json:
              schema:
                $ref: '#/components/schemas/undefined'  # <=
```

Example code which may generates this is: 
```js
    //                                                here----v---v
    imaged_hosts.entity_procedure('ping', {}, proc_strategy, {}, {}, ping_handler)
```

**Comment by shlomi-nx** [comment]
@Emulebest Please take a look, see whats missing. If nothing you can close.

**Comment by Emulebest** [comment]
resolved via #177
